packages/docusaurus-init/templates/classic/docs/doc1.md

```diff
@@ -166,3 +166,27 @@ Here's a line for us to start with.
 This line is separated from the one above by two newlines, so it will be a _separate paragraph_.
 
 This line is also a separate paragraph, but... This line is only separated by a single newline, so it's a separate line in the _same paragraph_.
+
+---
+
+## Admonitions
+
+:::note
+This is a note
+:::
+
+:::tip
+This is a tip
+:::
+
+:::important
+This is important
+:::
+
+:::caution
```

Comment:
I already have an alias of danger -> caution. I went with those words so that it is fully backwards compatible with `remarkable-admonitions` so that anyone upgrading their site will get the expected results. 

I based the ordering of cation warning danger on the guidelines provided here https://github.com/favoloso/remarkable-admonitions/issues/4

I can switch around the aliases (i.e. make it warning and danger with caution -> warning) for consistency with Infima over backwards compatibility.

Reply:
@yangshun which approach would you prefer?

Reply:
This was just a nit comment, it's fine as-is 😄